testsuite/integration-arquillian/test-apps/fuse/camel/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 Red Hat, Inc. and/or its affiliates
+  ~ and other contributors as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <artifactId>integration-arquillian-test-apps-fuse-parent</artifactId>
+        <groupId>org.keycloak.testsuite</groupId>
+        <version>999.0.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.keycloak.testsuite</groupId>
+    <artifactId>fuse-example-camel-endpoint</artifactId>
+    <packaging>bundle</packaging>
+    <name>Camel endpoint example - Secured in Karaf/Fuse</name>
+    <description/>
+
+    <properties>
+
+        <keycloak.osgi.export>
+        </keycloak.osgi.export>
+        <keycloak.osgi.import>
+            javax.servlet;version="[3.1,5)",
+            javax.servlet.http;version="[3.1,5)",
+            org.apache.camel.*,
+            org.apache.camel;version="[2.13,3)",
+            org.eclipse.jetty.security;version="[8,10)",
+            org.eclipse.jetty.server.nio;version="[8,10)",
+            org.eclipse.jetty.util.security;version="[8,10)",
+            org.keycloak.*;version="${project.version}",
```

Comment:
As far as I understand, we want this for backward compatibility of older adapters, should we change this to something like here? https://github.com/pdrozd/keycloak/blob/adapters-18x/testsuite/integration-arquillian/test-apps/spring-boot-adapter-app/pom.xml#L109 so we can change the version?

Reply:
Good catch Michal, thanks!